src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+module.exports = function(options, undef) {
+  'use strict';
+
+  const TAU = 2 * Math.PI,
+        //TAU = options.PConstants.TAU,
```

Comment:
We can get this value from `https://github.com/processing-js/processing-js/blob/master/src/Helpers/PConstants.js` by using:
```
var pConstants = require("../Helpers/PConstants"),
    TAU = pConstants.TAU;